knowhere/index/vector_index/IndexAnnoy.cpp

```diff
@@ -141,7 +143,11 @@ IndexAnnoy::Query(const DatasetPtr& dataset_ptr, const Config& config, const fai
     auto search_k = GetIndexParamSearchK(config);
     auto p_id = new int64_t[k * rows];
     auto p_dist = new float[k * rows];
-
+    if (CheckKeyInConfig(config, meta::QUERY_THREAD_NUM)) {
+        omp_set_num_threads(GetMetaQueryThreadNum(config));
```

Comment:
It set two parameters in meta,QUERY_THREAD_NUM and BUILD_THREAD_NUM, which refers to the parameters that user assigned. When users didn't assign it, the omp parallel section will use the default parameter seted in knowhere, DEFUALT_QUERY_THREAD_NUM and DEFUALT_BUILD_THREAD_NUM.